### PR TITLE
German language translation

### PR DESCRIPTION
**Pull request by Khazoda**
added lang file for german

### CODE REVIEW

File: src/main/resources/assets/someassemblyrequired/lang/de_de.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+  "block.someassemblyrequired.acacia_sandwich_assembly_table": "Akazien-Sandwich-Montagetisch",
+  "block.someassemblyrequired.birch_sandwich_assembly_table": "Birken-Sandwich-Montagetisch",
+  "block.someassemblyrequired.crimson_sandwich_assembly_table": "Purpurroter Sandwich-Montagetisch",
+  "block.someassemblyrequired.dark_oak_sandwich_assembly_table": "Dunkeleichen-Sandwich-Montagetisch",
+  "block.someassemblyrequired.jungle_sandwich_assembly_table": "Dschungel-Sandwich-Montagetisch",
+  "block.someassemblyrequired.oak_sandwich_assembly_table": "Eichen-Sandwich-Montagetisch",
+  "block.someassemblyrequired.spruce_sandwich_assembly_table": "Fichten-Sandwich-Montagetisch",
+  "block.someassemblyrequired.warped_sandwich_assembly_table": "Verzogener-Sandwich-Montagetisch",
+  "block.someassemblyrequired.acacia_cutting_board": "Akazien Schneidbrett",
```
Comment (Geolykt) [inline src/main/resources/assets/someassemblyrequired/lang/de_de.json:19]: Isn't it "Schneidebrett"?

File: src/main/resources/assets/someassemblyrequired/lang/de_de.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  "block.someassemblyrequired.dark_oak_cutting_board": "Dunkeleichen Schneidbrett",
+  "block.someassemblyrequired.jungle_cutting_board": "Dschungel Schneidbrett",
+  "block.someassemblyrequired.oak_cutting_board": "Eichen Schneidbrett",
+  "block.someassemblyrequired.spruce_cutting_board": "Fichten Schneidbrett",
+  "block.someassemblyrequired.warped_cutting_board": "Verzogener Schneidbrett",
+  "block.someassemblyrequired.redstone_toaster": "Redstone Toaster",
+  "block.someassemblyrequired.sticky_redstone_toaster": "Klebriger Redstone Toaster",
+  "block.someassemblyrequired.sandwich": "Sandwich",
+
+  "block.someassemblyrequired.sandwich_assembly_table.tooltip": "Can be used to make sandwiches by placing items on top. Sneak-right click with an empty hand to build your sandwich. Platziere hierauf deine zutaten zwischen brot, und dann SHIFT + Rechtsklick um dein Sandwich zu erschaffen",
```
Comment (Geolykt) [inline src/main/resources/assets/someassemblyrequired/lang/de_de.json:31]: The english translations still persist here

File: src/main/resources/assets/someassemblyrequired/lang/de_de.json [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  "item.someassemblyrequired.sandwich.toast": "Toast",
+  "item.someassemblyrequired.double_decker_sandwich": "Doppeldecker Sandwich",
+  "item.someassemblyrequired.single_sandwich": "%s Sandwich",
+  "item.someassemblyrequired.double_sandwich": "Doppeldecker %s Sandwich",
+  "item.someassemblyrequired.triple_sandwich": "Dreifachdecker %s Sandwich",
+  "item.someassemblyrequired.quadruple_sandwich": "Vierfachdecker %s Sandwich",
+  "item.someassemblyrequired.quintuple_sandwich": "Fünffachdecker %s Sandwich",
+  "item.someassemblyrequired.sextuple_sandwich": "Sechsfachdecker %s Sandwich",
+  "item.someassemblyrequired.septuple_sandwich": "Siebenfachdecker %s Sandwich",
+  "item.someassemblyrequired.very_large_sandwich": "Sehr Großes %s Sandwich",
```
Comment (Geolykt) [inline src/main/resources/assets/someassemblyrequired/lang/de_de.json:70]: I wouldn't capitalize "großes"

File: src/main/resources/assets/someassemblyrequired/lang/de_de.json [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  "item.someassemblyrequired.single_potion_sandwich": "Sandwich von %s",
+  "item.someassemblyrequired.double_potion_sandwich": "Doppeldecker Decker Sandwich von %s",
```
Comment (Geolykt) [inline src/main/resources/assets/someassemblyrequired/lang/de_de.json:72]: Duplicate "Decker"

File: src/main/resources/assets/someassemblyrequired/lang/de_de.json [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  "item.someassemblyrequired.septuple_potion_sandwich": "Siebenfachdecker Sandwich von %s",
+  "item.someassemblyrequired.very_large_potion_sandwich": "Sehr Großes Sandwich von %s",
+
+  "item.someassemblyrequired.sandwich.tooltip.truncate_info": "(%s zusätzliche Zutaten)",
+  "item.someassemblyrequired.kitchen_knife.tooltip": "Fürs Schneiden der Zutaten auf dem Schneidbrett",
+  "item.someassemblyrequired.spread.tooltip": "Wo fandest du das?",
+  "item.someassemblyrequired.bread_slice.tooltip": "Viel hungereffizienter als Brot und kann schneller gegessen werden",
+  "item.someassemblyrequired.toasted_bread_slice.tooltip": "Stellt mehr Hunger als eine Brotscheibe wieder her",
+  "item.someassemblyrequired.charred_bread_slice.tooltip": "Nicht ganz so lecker wie Brot"
+  "item.someassemblyrequired.apple_slices.tooltip": "Viel Hungereffizienter als ein ganzer Apfel und kann schneller gegessen werden",
```
Comment (Geolykt) [inline src/main/resources/assets/someassemblyrequired/lang/de_de.json:86]: Invalid JSON

(This is a critical issue)

File: src/main/resources/assets/someassemblyrequired/lang/de_de.json [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+  "item.someassemblyrequired.golden_apple_slices.tooltip": "Viel Hungereffizienter als ein ganzer goldener Apfel und kann schneller gegessen werden",
```
Comment (Geolykt) [inline src/main/resources/assets/someassemblyrequired/lang/de_de.json:87]: "Hungereffizienter" needn't be capitalized

File: src/main/resources/assets/someassemblyrequired/lang/de_de.json [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+  "item.someassemblyrequired.enchanted_golden_apple_slices.tooltip": "Viel hungereffizienter als ein ganzer verzauberter goldener Apfel und kann schneller gegessen werden",
+  "item.someassemblyrequired.chopped_carrot.tooltip": "Viel hungereffizienter als eine ganze Karotte und kann schneller gegessen werden",
+  "item.someassemblyrequired.chopped_golden_carrot.tooltip": "Viel hungereffizienter als eine ganze goldene Karotte und kann schneller gegessen werden",
+  "item.someassemblyrequired.chopped_beetroot.tooltip": "Viel hungereffizienter als eine ganze Rote Beete und kann schneller gegessen werden",
```
Comment (Geolykt) [inline src/main/resources/assets/someassemblyrequired/lang/de_de.json:91]: Isn't the "Rote" part not capitalized?

File: src/main/resources/assets/someassemblyrequired/lang/de_de.json [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+  "item.someassemblyrequired.double_decker_sandwich": "Doppeldecker Sandwich",
+  "item.someassemblyrequired.single_sandwich": "%s Sandwich",
+  "item.someassemblyrequired.double_sandwich": "Doppeldecker %s Sandwich",
+  "item.someassemblyrequired.triple_sandwich": "Dreifachdecker %s Sandwich",
+  "item.someassemblyrequired.quadruple_sandwich": "Vierfachdecker %s Sandwich",
+  "item.someassemblyrequired.quintuple_sandwich": "Fünffachdecker %s Sandwich",
+  "item.someassemblyrequired.sextuple_sandwich": "Sechsfachdecker %s Sandwich",
+  "item.someassemblyrequired.septuple_sandwich": "Siebenfachdecker %s Sandwich",
+  "item.someassemblyrequired.very_large_sandwich": "Sehr Großes %s Sandwich",
+  "item.someassemblyrequired.single_potion_sandwich": "Sandwich von %s",
```
Comment (Geolykt) [inline src/main/resources/assets/someassemblyrequired/lang/de_de.json:71]: If I'm not wrong the better translation would be
"Sandwich der %s"

**Review by Geolykt — CHANGES_REQUESTED** [review]
Contains a critical issue (won't work) and several minor ones.

File: src/main/resources/assets/someassemblyrequired/lang/de_de.json (hunk 1 above)
Comment (Khazoda) [reply src/main/resources/assets/someassemblyrequired/lang/de_de.json:19]: thanks for all the suggestions, was pretty tired when doing this last night 😄 . Agreed it is Schneidebrett.

**Comment by ochotonida** [comment]
Thanks, I'll add this next update!

**Pull request merged**